### ISSUE TITLE
No sound after update

### ISSUE THREAD

**Issue by FreeWind6** [immich-app/immich#14653]
### The bug

After the update the latest version, I noticed that the videos made on the iPhone 15 ProMax, on Android, there is no sound on these videos. Tell me, is this a known problem?
Version where problem not found: 1.121.0

### The OS that Immich Server is running on

TrueNas

### Version of Immich Server

v1.122.2

### Version of Immich Mobile App

latest

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
TrueNas
```

### Your .env content

```Shell
TrueNas
```

### Reproduction steps

1. Upload video from ios
2. Watch video to android

**Comment by akostadinov** [comment]
I assume it will be helpful to post a good and bad video here so immich team can investigate. It is likely highly related to your phone model (this doesn't mean the videos should not be supported).

**Comment by FreeWind6** [comment]
Phone: iPhone 11
1) [Photo mode (bug)](https://github.com/user-attachments/assets/de641115-cf2e-42b7-ad08-32edfcbb3138)
2) [Video mode (not bug)](https://github.com/user-attachments/assets/eebe9e65-0bfe-4962-8c55-112bfe0dbe50)

Steps:
1) Download 1 video and upload to immich
2) Open App Immich for Andoid device
3) No sound in video
4) Open video on iOS device or windows device - sound yes

**Comment by csuno** [comment]
any news, on this?

We have same Problem on Iphone 11. And on some videos there is no sound on android app.

Immich server: Docker, 1.129

**Comment by 0xadam** [comment]
Hi all,

I hit this issue (thinking it was just a general iPhone video issue) and just managed to resolve it. Thanks for your videos @FreeWind6 - using them I was able to figure out the issue is the PCM audio in the QuickTake videos and the default mp4 container immich uses for transcoded videos.

### TL;DR 
By default in Immich PCM audio is copied as is to transcoded videos which are stored in an mp4 container. The mp4 format doesn't strictly support PCM, and so some places don't play it back i.e. the Immich Android app. It can be fixed by forcing transcoding of audio that is PCM. From the Immich web interface, click the profile button in the top right and select Administration. Navigate to Settings -> Video Transcoding Settings -> Transcode Policy, and under "Accepted audio codecs" uncheck PCM (16 bit). Then transcode affected videos again (I just did all videos using Jobs -> Transcode Videos -> All).

### The Details

Looking at the videos uploaded above by @FreeWind6, I found my Android Immich app would also not play audio on the one with the bug (a QuickTake video) by default, unless I turned on "Force Original Video" in the Asset Viewer section of the apps Settings. I downloaded the transcoded videos for both the working and non-working audio cases and used the mediainfo tool to diff the video metadata, and I noticed audio wise that the working video had `AAC LC` as the format, while the non-working video had `PCM` as the format. The same audio difference was present on the non-transcoded videos too. Interestingly however the PCM audio in the non-transcoded video had a Codec ID of "lpcm", while it was "ipcm" in the transcoded one. Googling around ipcm lead me to the fact that PCM audio isn't typically stored in mp4 files[1][2], and that is what Immich stores videos in by default (at least in my instance on 1.134.0) so I figured maybe that was the issue. Further to this, in the Immich Transcode Policy "Accepted audio codecs" (See TLDR above for where this is) it defaults to not transcoding audio that is "PCM (16 bit)", creating the issue. 

I chose to simply uncheck PCM (16 bit) to force transcoding in the future, and after retranscoding that fixed the issue for both the video uploaded here and one of the videos I was having issues with (from the Immich web view of a video with no audio I selected "Refresh encoded videos" from the 3 dots menu to retranscode). However maybe there is a way to force Immich to not use mp4 and pick some other container that supports PCM audio if wanting to keep the original audio? Though I am not sure... (looking at "Accepted containers" under the "Transcode Policy" section my read is it should be keeping the mov container used by QuickTake videos anyway as MOV is by default an accepted container?)

Hopefully this is useful for someone else too :)

[1] https://www.reddit.com/r/shutterencoder/comments/1ioowxu/comment/mdiduok/
[2] https://community.adobe.com/t5/after-effects-discussions/adobe-media-encoder-mp4-container-with-pcm-audio/m-p/9030844#M27660

**Comment by FreeWind6** [comment]
@0xadam thank you very much!!! This actually solved my problem!!! Cool!!
